compendium/ConfigurationAdmin/src/CMLogger.cpp

```diff
@@ -36,8 +36,11 @@ CMLogger::CMLogger(cppmicroservices::BundleContext context)
 
 CMLogger::~CMLogger()
 {
-  if (serviceTracker) {
-    serviceTracker->Close();
+  try {
+    if (serviceTracker) {
+      serviceTracker->Close();
+    }
+  } catch (...) {
   }
```

Comment:
Should we issue a warning to the log an exception gets thrown rather than just swallowing it?

Reply:
There is no logger at this point to log to since it would have been removed on `serviceTracker->Close()`.

I can refactor `CMLogger` similar to `SCRLogger` to close the service tracker outside of the destructor to be able to log exceptions. I would prefer to make that change separate to this PR. This change is only necessary to fix a crash in the tests.